### PR TITLE
Feature/change client tools dir name

### PR DESCRIPTION
**Pull request by cpjulia**
### Scope & Purpose

Accompanying Oskar PR: https://github.com/arangodb/oskar/pull/358

Changed arangosh directory name to client-tools, because it inferred an understanding that this directory was only about arangosh, and not all the other client tools.

- [ ] :hankey: Bugfix (requires CHANGELOG entry)
- [x] :pizza: New feature (requires CHANGELOG entry, feature documentation and release notes)
- [ ] :fire: Performance improvement
- [ ] :hammer: Refactoring/simplification
- [x] :book: CHANGELOG entry made

#### Backports:

No backports required.

#### Related Information

Accompanying Oskar PR: https://github.com/arangodb/oskar/pull/358

### Testing & Verification

*(Please pick either of the following options)*

- [ ] This change is a trivial rework / code cleanup without any test coverage.
- [x] The behavior in this PR was *manually tested*
- [ ] This change is already covered by existing tests, such as *(please describe tests)*.
- [ ] This PR adds tests that were used to verify all changes:
  - [ ] Added new C++ **Unit tests**
  - [ ] Added new **integration tests** (e.g. in shell_server / shell_server_aql)
  - [ ] Added new **resilience tests** (only if the feature is impacted by failovers)
- [ ] There are tests in an external testing repository:
- [ ] I ensured this code runs with ASan / TSan or other static verification tools

Jenkins runs:
https://jenkins.arangodb.biz:3456/view/PR/job/arangodb-matrix-pr/18243/

https://jenkins.arangodb.biz:3456/job/arangodb-ANY-packages/46/

### CODE REVIEW

File: 3rdParty/velocypack/tests/jsonSample/directory-tree.json [hunk 1]
```diff
@@ -1 +1 @@
...ries":{"internal":{"name":"internal","files":[{"name":"gmock-internal-utils.h","size":20374},{"name":"gmock-generated-internal-utils.h","size":11423},{"name":"gmock-port.h","size":3974}]}}}}}}},"gtest":{"name":"gtest","directories":{"samples":{"name":"samples","files":[{"name":"sample3_unittest.cc","size":5351},{"name":"sample10_unittest.cc","size":5053},{"name":"sample4.h","size":2083},{"name":"sample1.cc","size":2502},{"name":"sample4_unittest.cc","size":1909},{"name":"sample3-inl.h","size":5365},{"name":"sample8_unittest.cc","size":6941},{"name":"sample2.cc","size":2298},{"name":"sample2_unittest.cc","size":3926},{"name":"sample7_unittest.cc","size":5099},{"name":"sample2.h","size":3006},{"name":"sample6_unittest.cc","size":8989},{"name":"sample5_unittest.cc","size":6590},{"name":"sample9_unittest.cc","size":5951},{"name":"sample1_unittest.cc","size":5129},{"name":"sample4.cc","size":1927},{"name":"sample1.h","size":1937},{"name":"prime_tables.h","size":4072}]},"test":{"name":"test","files":[{"name":"gtest-test-part_test.cc","size":7282},{"name":"gtest-typed-test_test.cc","size":11370},{"name":"production.h","size":2172},{"name":"gtest_help_test_.cc","size":2131},{"name":"gtest_
```
Comment (jsteemann) [inline 3rdParty/velocypack/tests/jsonSample/directory-tree.json:1]: We should not have any renaming in here. This is an unrelated file.

File: 3rdParty/velocypack/tests/jsonSample/file-list.json [hunk 2]
```diff
@@ -1 +1 @@
...nCluster.h","ApplicationCluster.cpp","RestShardHandler.cpp","ServerState.cpp","RestShardHandler.h","ServerState.h","ClusterComm.h","ServerJob.h","ClusterInfo.h","ClusterComm.cpp","ClusterMethods.h","ClusterInfo.cpp","ClusterMethods.cpp","AgencyComm.h","HeartbeatThread.cpp","ServerJob.cpp"]},"HttpServer":{"name":"HttpServer","files":["HttpsCommTask.h","ApplicationEndpointServer.cpp","HttpServerJob.h","HttpListenTask.h","HttpHandlerFactory.h","AsyncJobManager.h","HttpHandler.cpp","PathHandler.h","HttpsCommTask.cpp","HttpListenTask.cpp","HttpServer.h","HttpHandler.h","HttpServer.cpp","HttpServerJob.cpp","PathHandler.cpp","HttpsServer.cpp","AsyncJobManager.cpp","HttpCommTask.cpp","ApplicationEndpointServer.h","HttpCommTask.h","HttpHandlerFactory.cpp","HttpsServer.h"]},"Dispatcher":{"name":"Dispatcher","files":["DispatcherThread.h","RequeueTask.h","Job.cpp","DispatcherThread.cpp","Dispatcher.cpp","DispatcherQueue.h","Job.h","ApplicationDispatcher.cpp","RequeueTask.cpp","DispatcherQueue.cpp","Dispatcher.h","ApplicationDispatcher.h"]},"Replication":{"name":"Replication","files":["InitialSyncer.cpp","Syncer.h","ContinuousSyncer.cpp","Syncer.cpp","ContinuousSyncer.h","InitialSyncer.h"]}}}}}
```
Comment (jsteemann) [inline 3rdParty/velocypack/tests/jsonSample/file-list.json:1]: We should not have any renaming in here. This is an unrelated file.

File: CHANGELOG [hunk 3]
```diff
@@ -1,6 +1,8 @@
 devel
 -----
 
+* Changed arangosh directory name to client-tools
```
Comment (jsteemann) [inline CHANGELOG:4]:
```suggestion
* Changed "arangosh" directory name to "client-tools", because the directory contains
  the code for all client tools and not just arangosh.
```

**Review by dothebart — APPROVED** [review]
please make sure (by testing in the CI) that documentation generation still works with this PR.

**Comment by joerg84** [comment]
@KVS85 @cpjulia Can this PR be merged?

**Comment by cpjulia** [comment]
> 

I think we should test it some more

**Comment by KVS85** [comment]
- [ ] https://jenkins.arangodb.biz/view/Documentation/job/arangodb-ANY-examples/1315/
- [ ] https://jenkins.arangodb.biz/job/arangodb-documentation-ng2/1/

File: CHANGELOG [hunk 4]
```diff
@@ -1,6 +1,11 @@
 devel
 -----
 
+* Changed "arangosh" directory name to "client-tools", because the directory 
+  contains the code for all client tools and not just arangosh.
+
+* Make the `--version` and `--version-json` commands usable in arangobackup
+
```
Comment (jsteemann) [inline CHANGELOG:8]:
```suggestion
```

**Review by jsteemann — APPROVED** [review]
LGTM!
